h/static/styles/annotations.scss

```diff
@@ -34,7 +34,14 @@
   margin: .8em 0;
 }
 
-.annotation-header { margin-top: 0 }
+.annotation-header {
+  margin-top: 0;
+
+  &.please-sign-in {
```

Comment:
Have you looked at removing or changing existing styles instead of adding this special case?